doc/backlog/io.md

```
@@ -1,5 +1,17 @@
 # Backlog for `ctk {load,save} [table]`
 
+## General
+- ingestr: Educate about batch size (`page_size`)
+- ingestr: More parameters for Kinesis
+  - https://getbruin.com/docs/ingestr/commands/ingest.html
+  - https://getbruin.com/docs/ingestr/supported-sources/kinesis.html
+  - ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)